.azure-pipelines/simplified_PR_test/README.md

```diff
@@ -0,0 +1,106 @@
+## Background
+In current PR testing process, a fixed set of test scripts is executed regardless of the change scope.
+This approach lacks flexibility. On the one hand, if changes are only related to a few lines of codebase,
+we may don't need to run the whole scope. On the other hand, if there are new added test scripts,
+we need to add them manually.
+
+With approximately 570 test scripts running, the process has become excessively large and the runtime increased significantly.
+Due to the maximum execution time limit, more instances are needed to run the tests in parallel.
+For example, to meet this requirement, we need 20 instances for t0 and 25 instances for t1.
+The cost per PR has reached $35, and we will use $23,000 per month to run PR testing, which is considerably high.
+
+To address these issues, we propose a new PR testing model called 'Impacted Area-Based PR Testing.
+
+## Preparation
+We can organize the codebase in this way:
+```
+sonic-mgmgt
+     | - .azure-pipelines
+     | - ansible
+     | - docs
+     | - ......
+     | - tests
+           |
+           | - common      ---------- shared
+           | - arp         -----|
+           | - ecmp             | --- features
+           | - vlan             |
+           | - ......      -----|
+```
+Under sonic-mgmt, there are several top-level folders such as `.azure-pipelines`, `ansible`, `docs`, `tests`, and more.
+Except for the `tests` folder, we classify all other folders as part of the shared section of the repo.
+
+Within the `tests` folder, there are multiple second-level directories.
+Among them, the common folder is also considered part of the shared section.
+Other folders, such as `arp`, `ecmp`, and similar directories, are classified as feature-specific parts.
+
+Scripts in the common folder fall under the shared section and can be utilized across different folders.
+In contrast, scripts in other folders belong to the features section, representing specific functionalities such as arp, ecmp, and vlan,
+and are intended for use within their respective folders.
+This hierarchy helps us more effectively identify the impacted areas for the new PR testing process.
+
+However, the previous code had numerous cross-feature dependencies.
+To achieve our goal, we carried out some preparatory work by eliminating these cross-feature dependencies.
+
+
+## Design
```

Comment:
Can you add some chart for better understan the work flow?

Reply:
The design flowchart, step by step

Reply:
I think each step is listed as a subtitle in the part "Design". Here, we can not see the process dynamically, so I don't think it's necessary to add the flowchart.